sdk/storage/azure-storage-blob/azure/storage/blob/_shared/parser.py

```diff
@@ -18,3 +18,10 @@ def _str(value):
 
 def _to_utc_datetime(value):
     return value.strftime('%Y-%m-%dT%H:%M:%SZ')
+
+
+def get_empty_chunk(chunk_size):
```

Comment:
We might want to move this method under blob\_deserialize.py if you are good with the second commit(which is to remove the download from _shared fold) *^_^*